package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "cast-monorepo",
+  "version": "0.1.0",
+  "description": "A monorepo for the Cast app",
+  "private": true,
+  "scripts": {
+    "preinstall": "npx only-allow pnpm"
+  },
+  "repository": {
```

Comment:
[praise](https://conventionalcomments.org/#labels): this is quite an extensive `package.json`, cool :)